examples/all-clusters-app/esp32/partitions.csv

```diff
@@ -1,8 +1,11 @@
 # Name,   Type, SubType, Offset,  Size, Flags
 # Note: if you have increased the bootloader size, make sure to update the offsets to avoid overlap
-nvs,      data, nvs,     ,        0x6000,
-otadata,  data, ota,     ,        0x2000,
-phy_init, data, phy,     ,        0x1000,
-ota_0,    app,  ota_0,   ,        1700K,
-ota_1,    app,  ota_1,   ,        1700K,
-ot_storage, data, 0x3a,            , 0x2000,
+dac_prv_key, data, 0xFF,  0xb000,  0x1000, encrypted,
```

Comment:
Let's keep the example as simple as it is, but enhance the logic to support certs relocation.